.github/workflows/i18n.yml

```diff
@@ -0,0 +1,135 @@
+---
+name: I18n
+on:
+  schedule:
+    - cron: '0 7 * * 0,4'
+
+env:
+  JAVA_DISTRIBUTION: 'temurin'
+
+jobs:
+  main_branch_i18n:
+    name: main branch i18n
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:38
+
+    steps:
+      - name: Mask secrets
+        shell: bash
+        run: |
+          echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Install dependencies
+        shell: bash
+        run: dnf --setopt install_weak_deps=False install -y gettext jss git
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: main
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+          java-version: 17
+
+      - name: Add safe directory
+        shell: bash
+        run: git config --system --add safe.directory /__w/*
+
+      - name: Run internationlization
+        shell: bash
+        run: |
+          chmod +x ./.github/scripts/internationalization.sh
+          ./.github/scripts/internationalization.sh
+        env:
+          GIT_AUTHOR_EMAIL: '3229038+candlepin-bot@users.noreply.github.com'
+          GIT_AUTHOR_NAME: 'candlepin-bot'
+          GIT_BRANCH: 'main'
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  4.1_branch_i18n:
```

Comment:
I thought this would be done by simply adding the workflow in each separate branch, but it turns out scheduled workflows only run on the default branch (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule). So I guess this is the only way of doing this